docs/dev-env-first-time-contributors.md

```diff
@@ -982,6 +982,9 @@ Now run `vagrant up` in your terminal to install the development
 server. If you ran `vagrant up` before and failed, you'll need to run
 `vagrant destroy` first to clean up the failed installation.
 
+**If you want to remove proxy from Vagrant delete `~/.zulip-vagrant-config`
+file and restart Vagrant.**
```

Comment:
This should say "If you no longer want to use the proxy with Vagrant, delete ..." for clearer English.